### PR TITLE
Public Release of Commonalities 0.4.0

### LINKED ISSUE #175: Commonalities release 0.4.0 scope
**Problem description**
The next release of Commonalities should be the base for Fall24 CAMARA meta-release.

**Expected action**
The **M0** milestone (scope definition of the release should be achieved in the next Commonalities meeting on 15/04 

**Additional context**

The timeline for Fall24 - https://wiki.camaraproject.org/display/CAM/Meta-release+Fall24
image-2024-4-4_16-7-4

### PR DESCRIPTION
**Pull request by rartych**
#### What type of PR is this?

* subproject management

#### What this PR does / why we need it:

This is Release PR for public release

#### Which issue(s) this PR fixes:

Fixes #175 #282 

#### Changelog input

```
Commonalities Public Release 0.4.0

```

### CODE REVIEW

**Review by PedroDiez — DISMISSED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ NOTE: Guidelines for Release Management of API versions, e.g. the API-Readiness-
 
 ## Status and released versions
 
-* Release 0.4.0-rc.2 of guidelines and assets for CAMARA APIs is available with the [r0.4.0-rc.2 tag](https://github.com/camaraproject/Commonalities/tree/r0.4.0-rc.2):
+* Release 0.4.0 of guidelines and assets for CAMARA APIs is available with the [r0.4.0 tag](https://github.com/camaraproject/Commonalities/tree/r0.4.0):
 * Release 0.3.0 of guidelines and assets for CAMARA APIs is available within the [release-0.3.0 branch](https://github.com/camaraproject/Commonalities/tree/release-0.3.0):
 * Release 0.2.0 of guidelines and assets for CAMARA APIs is available within the [release-0.2.0 branch](https://github.com/camaraproject/Commonalities/tree/release-0.2.0):
 * Release 0.1.0 of guidelines and assets for CAMARA APIs is available within the [release-0.1.0 branch](https://github.com/camaraproject/Commonalities/tree/release-0.1.0):
```
Comment (hdamker) [inline README.md:47]: I'm not sure if we need all the previous releases here as we get now a new latest release. Especially the older one might be just confusing for the audience without seeing also the changelog.

How about:

```suggestion
* Release 0.4.0 of guidelines and assets for CAMARA APIs is available with the [r0.4.0 tag](https://github.com/camaraproject/Commonalities/tree/r0.4.0):
* Previous releases and pre-releases are available in https://github.com/camaraproject/Commonalities/releases
```

**Review by PedroDiez — APPROVED** [review]
LGTM

**Review by shilpa-padgaonkar — APPROVED** [review]
/LGTM

**Review by hdamker — APPROVED** [review]
Approved on behalf of Release Management and the TSC (cf. https://wiki.camaraproject.org/x/pYBFAg).

**Comment by hdamker** [comment]
@rartych would you merge and create the release?

**Comment by rartych** [comment]
Merging as approved by TSC and Release Management

**Pull request merged**